client/src/Pages/Teacher/MyQuizzes/DeleteQuizModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
Change `MyQuizzes` folder name to `Quizzes`, because it's inside the `Teacher` folder it makes more sense to be Quizzes